### ISSUE TITLE
[Polish] a11y punchlist

### ISSUE THREAD

**Issue by clatwell** [Good-Bids/goodbids#646]
Now that we have a11y Checker Pro working properly on Staging (!!), we'd like to take a quick pass through the reports for the GB Main Site and a Nonprofit Site with default settings / content and see if we can resolve some of the Warnings so that the sites can start fresh. It looks like we're in great shape with no Errors (aside from those introduced by Users, ie. no alt text or missing transcript for videos)  

https://staging.goodbids.org/wp-admin/admin.php?page=accessibility_checker 
https://staging.goodbids.org/charity-water/wp-admin/admin.php?page=accessibility_checker_issues

### GB Main Site 

- WARNING: Resolve ARIA hidden issue with w3 svg -- it looks like this is showing up on most / all pages 

![Image](https://github.com/Good-Bids/goodbids/assets/16839493/517f9737-5409-407e-aa00-f599186fe5b9)

- WARNING: Resolve ARIA hidden issue with Jetpack feedback form elements 
- Adjust level of h element for site directory block 

![Image](https://github.com/Good-Bids/goodbids/assets/16839493/1b229183-61b8-468a-a4f3-418192c59a27)

### Nonprofit Site 

- Resolve ARIA hidden issue with w3 svg 
- Adjust level of h element for comment block 

![Image](https://github.com/Good-Bids/goodbids/assets/16839493/96566ade-f1dd-4460-a41c-3814e054eec5)
